In [1]:
import pandas as pd
pd.set_option('display.max_columns', 500)
pd.set_option('display.max_rows', 500)

In [9]:
df = pd.read_csv('despesas_pagas_candidatos_2018_MG.csv', sep=';',
          encoding='latin1', decimal=',', parse_dates=[0])
df.head().T

0  \
DT_GERACAO                                       2019-08-03 00:00:00   
HH_GERACAO                                                  23:41:02   
ANO_ELEICAO                                                     2018   
CD_TIPO_ELEICAO                                                    2   
NM_TIPO_ELEICAO                                            Ordinária   
CD_ELEICAO                                                       297   
DS_ELEICAO                                Eleição Geral Federal 2018   
DT_ELEICAO                                                07/10/2018   
ST_TURNO                                                           1   
TP_PRESTACAO_CONTAS                                          Parcial   
DT_PRESTACAO_CONTAS                                       13/09/2018   
SQ_PRESTADOR_CONTAS                                        424053973   
SG_UF                                                             MG   
DS_TIPO_DOCUMENTO                                              Outro   
NR_DOCUMENTO                                                  850003   
CD_FONTE_DESPESA                                                   2   
DS_FONTE_DESPESA         Fundo Especial de Financiamento de Campanha   
CD_ORIGEM_DESPESA                                           20040000   
DS_ORIGEM_DESPESA                     Locação/cessão de bens imóveis   
CD_NATUREZA_DESPESA                                                1   
DS_NATUREZA_DESPESA                                       Financeiro   
CD_ESPECIE_RECURSO                                                 1   
DS_ESPECIE_RECURSO                                        Financeiro   
SQ_DESPESA                                                  18055868   
SQ_PARCELAMENTO_DESPESA                                     10727673   
DT_PAGTO_DESPESA                                          06/09/2018   
DS_DESPESA                            Locação/cessão de bens imóveis   
VR_PAGTO_DESPESA                                                 660   

                                                                   1  \
DT_GERACAO                                       2019-08-03 00:00:00   
HH_GERACAO                                                  23:41:02   
ANO_ELEICAO                                                     2018   
CD_TIPO_ELEICAO                                                    2   
NM_TIPO_ELEICAO                                            Ordinária   
CD_ELEICAO                                                       297   
DS_ELEICAO                                Eleição Geral Federal 2018   
DT_ELEICAO                                                07/10/2018   
ST_TURNO                                                           1   
TP_PRESTACAO_CONTAS                                          Parcial   
DT_PRESTACAO_CONTAS                                       13/09/2018   
SQ_PRESTADOR_CONTAS                                        424053973   
SG_UF                                                             MG   
DS_TIPO_DOCUMENTO                                        Nota Fiscal   
NR_DOCUMENTO                                                  850002   
CD_FONTE_DESPESA                                                   2   
DS_FONTE_DESPESA         Fundo Especial de Financiamento de Campanha   
CD_ORIGEM_DESPESA                                           20140000   
DS_ORIGEM_DESPESA                Publicidade por materiais impressos   
CD_NATUREZA_DESPESA                                                1   
DS_NATUREZA_DESPESA                                       Financeiro   
CD_ESPECIE_RECURSO                                                 1   
DS_ESPECIE_RECURSO                                        Financeiro   
SQ_DESPESA                                                  18055869   
SQ_PARCELAMENTO_DESPESA                                     10727674   
DT_PAGTO_DESPESA                                          06/09/2018   
DS_DESPESA                       P

In [2]:
df1 = pd.read_csv('despesas_contratadas_candidatos_2018_MG.csv',
                 sep=';', encoding='latin1', decimal=',', parse_dates=[0])

df1.head().T

0  \
DT_GERACAO                                            2019-09-03 00:00:00   
HH_GERACAO                                                       00:55:37   
ANO_ELEICAO                                                          2018   
CD_TIPO_ELEICAO                                                         2   
NM_TIPO_ELEICAO                                                 Ordinária   
CD_ELEICAO                                                            297   
DS_ELEICAO                                     Eleição Geral Federal 2018   
DT_ELEICAO                                                     07/10/2018   
ST_TURNO                                                                1   
TP_PRESTACAO_CONTAS                                                 Final   
DT_PRESTACAO_CONTAS                                            14/01/2019   
SQ_PRESTADOR_CONTAS                                             427412898   
SG_UF                                                                  MG   
SG_UE                                                                  MG   
NM_UE                                                        MINAS GERAIS   
NR_CNPJ_PRESTADOR_CONTA                                    31243399000165   
CD_CARGO                                                                6   
DS_CARGO                                                 Deputado Federal   
SQ_CANDIDATO                                                 130000626675   
NR_CANDIDATO                                                         3130   
NM_CANDIDATO                                JUSSARA MENICUCCI DE OLIVEIRA   
NR_CPF_CANDIDATO                                              41352572672   
NR_CPF_VICE_CANDIDATO                                                  -1   
NR_PARTIDO                                                             31   
SG_PARTIDO                                                            PHS   
NM_PARTIDO                             Partido Humanista da Solidariedade   
CD_TIPO_FORNECEDOR                                                      0   
DS_TIPO_FORNECEDOR                                          Pessoa Física   
CD_CNAE_FORNECEDOR                                                     -1   
DS_CNAE_FORNECEDOR                                                 #NULO#   
NR_CPF_CNPJ_FORNECEDOR                                        61377155668   
NM_FORNECEDOR                                       IMARA TEIXEIRA GARCIA   
NM_FORNECEDOR_RFB                                   IMARA TEIXEIRA GARCIA   
CD_ESFERA_PART_FORNECEDOR                                              -1   
DS_ESFERA_PART_FORNECEDOR                                          #NULO#   
SG_UF_FORNECEDOR                                                   #NULO#   
CD_MUNICIPIO_FORNECEDOR                                                -1   
NM_MUNICIPIO_FORNECEDOR                                            #NULO#   
SQ_CANDIDATO_FORNECEDOR                                                -1   
NR_CANDIDATO_FORNECEDOR                                                -1   
CD_CARGO_FORNECEDOR                                                    -1   
DS_CARGO_FORNECEDOR                                                #NULO#   
NR_PARTIDO_FORNECEDOR                                                  -1   
SG_PARTIDO_FORNECEDOR                                              #NULO#   
NM_PARTIDO_FORNECEDOR                                              #NULO#   
DS_TIPO_DOCUMENTO                                                   Outro   
NR_DOCUMENTO                                                       092018   
CD_ORIGEM_DESPESA                                                20010000   
DS_ORIGEM_DESPESA                                    Despesas com pessoal   
SQ_DESPESA                                                       24828857   
DT_DESPESA                                                     07/09/2018   
DS_DESPESA                 SERVIÇOS NA CAMPANHA ELEITORAL. CABO ELEI

In [8]:
df1[df1["SQ_CANDIDATO"]==130000624468].shape

(65, 53)

In [10]:
df[df["SQ_PRESTADOR_CONTAS"]==427249064].shape

(60, 28)

In [13]:
df[df["SQ_PRESTADOR_CONTAS"]==427249064].sort_values(by=["VR_PAGTO_DESPESA"],ascending=0).T

86951  \
DT_GERACAO                      2019-08-03 00:00:00   
HH_GERACAO                                 23:41:02   
ANO_ELEICAO                                    2018   
CD_TIPO_ELEICAO                                   2   
NM_TIPO_ELEICAO                           Ordinária   
CD_ELEICAO                                      297   
DS_ELEICAO               Eleição Geral Federal 2018   
DT_ELEICAO                               07/10/2018   
ST_TURNO                                          1   
TP_PRESTACAO_CONTAS                           Final   
DT_PRESTACAO_CONTAS                      06/11/2018   
SQ_PRESTADOR_CONTAS                       427249064   
SG_UF                                            MG   
DS_TIPO_DOCUMENTO                       Nota Fiscal   
NR_DOCUMENTO                                 900007   
CD_FONTE_DESPESA                                  0   
DS_FONTE_DESPESA                   Fundo Partidário   
CD_ORIGEM_DESPESA                          20110000   
DS_ORIGEM_DESPESA          Publicidade por adesivos   
CD_NATUREZA_DESPESA                               1   
DS_NATUREZA_DESPESA                      Financeiro   
CD_ESPECIE_RECURSO                                1   
DS_ESPECIE_RECURSO                       Financeiro   
SQ_DESPESA                                 22419789   
SQ_PARCELAMENTO_DESPESA                    14462897   
DT_PAGTO_DESPESA                         20/09/2018   
DS_DESPESA                 Publicidade por adesivos   
VR_PAGTO_DESPESA                            61715.5   

                                                       86969  \
DT_GERACAO                               2019-08-03 00:00:00   
HH_GERACAO                                          23:41:02   
ANO_ELEICAO                                             2018   
CD_TIPO_ELEICAO                                            2   
NM_TIPO_ELEICAO                                    Ordinária   
CD_ELEICAO                                               297   
DS_ELEICAO                        Eleição Geral Federal 2018   
DT_ELEICAO                                        07/10/2018   
ST_TURNO                                                   1   
TP_PRESTACAO_CONTAS                                    Final   
DT_PRESTACAO_CONTAS                               06/11/2018   
SQ_PRESTADOR_CONTAS                                427249064   
SG_UF                                                     MG   
DS_TIPO_DOCUMENTO                                Nota Fiscal   
NR_DOCUMENTO                                          900003   
CD_FONTE_DESPESA                                           0   
DS_FONTE_DESPESA                            Fundo Partidário   
CD_ORIGEM_DESPESA                                   20140000   
DS_ORIGEM_DESPESA        Publicidade por materiais impressos   
CD_NATUREZA_DESPESA                                        1   
DS_NATUREZA_DESPESA                               Financeiro   
CD_ESPECIE_RECURSO                                         1   
DS_ESPECIE_RECURSO                                Financeiro   
SQ_DESPESA                                          22419794   
SQ_PARCELAMENTO_DESPESA                             14462887   
DT_PAGTO_DESPESA                                  18/09/2018   
DS_DESPESA               Publicidade por materiais impressos   
VR_PAGTO_DESPESA                                       45790   

                                                       86934  \
DT_GERACAO                               2019-08-03 00:00:00   
HH_GERACAO                                          23:41:02   
ANO_ELEICAO                                             2018   
CD_TIPO_ELEICAO                                            2   
NM_TIPO_ELEICAO                                    Ordinária   
CD_ELEICAO                                               297   
DS_ELEICAO                        Eleição Geral Federal 2018   
DT_ELEICAO                                        07/10/2018   
ST_TURNO         

In [7]:
df1[df1["SQ_CANDIDATO"]==130000624468].sort_values(by=["VR_DESPESA_CONTRATADA"],ascending=0).T

14172  \
DT_GERACAO                                               2019-09-03 00:00:00   
HH_GERACAO                                                          00:55:37   
ANO_ELEICAO                                                             2018   
CD_TIPO_ELEICAO                                                            2   
NM_TIPO_ELEICAO                                                    Ordinária   
CD_ELEICAO                                                               297   
DS_ELEICAO                                        Eleição Geral Federal 2018   
DT_ELEICAO                                                        07/10/2018   
ST_TURNO                                                                   1   
TP_PRESTACAO_CONTAS                                                    Final   
DT_PRESTACAO_CONTAS                                               06/11/2018   
SQ_PRESTADOR_CONTAS                                                427249064   
SG_UF                                                                     MG   
SG_UE                                                                     MG   
NM_UE                                                           MINAS GERAIS   
NR_CNPJ_PRESTADOR_CONTA                                       31234261000108   
CD_CARGO                                                                   6   
DS_CARGO                                                    Deputado Federal   
SQ_CANDIDATO                                                    130000624468   
NR_CANDIDATO                                                            1913   
NM_CANDIDATO                             JULIARINA APARECIDA DA CRUZ PEREIRA   
NR_CPF_CANDIDATO                                                  4143283611   
NR_CPF_VICE_CANDIDATO                                                     -1   
NR_PARTIDO                                                                19   
SG_PARTIDO                                                              PODE   
NM_PARTIDO                                                           Podemos   
CD_TIPO_FORNECEDOR                                                         1   
DS_TIPO_FORNECEDOR                                           Pessoa Jurídica   
CD_CNAE_FORNECEDOR                                                     18113   
DS_CNAE_FORNECEDOR         Impressão de jornais, livros, revistas e outra...   
NR_CPF_CNPJ_FORNECEDOR                                        19270206000160   
NM_FORNECEDOR                                              RONA EDITORA LTDA   
NM_FORNECEDOR_RFB                                          RONA EDITORA LTDA   
CD_ESFERA_PART_FORNECEDOR                                                 -1   
DS_ESFERA_PART_FORNECEDOR                                             #NULO#   
SG_UF_FORNECEDOR                                                      #NULO#   
CD_MUNICIPIO_FORNECEDOR                                                   -1   
NM_MUNICIPIO_FORNECEDOR                                               #NULO#   
SQ_CANDIDATO_FORNECEDOR                                                   -1   
NR_CANDIDATO_FORNECEDOR                                                   -1   
CD_CARGO_FORNECEDOR                                                       -1   
DS_CARGO_FORNECEDOR                                                   #NULO#   
NR_PARTIDO_FORNECEDOR                                                     -1   
SG_PARTIDO_FORNECEDOR                                                 #NULO#   
NM_PARTIDO_FORNECEDOR                                                 #NULO#   
DS_TIPO_DOCUMENTO                                                Nota Fiscal   
NR_DOCUMENTO                                                         641 - E   
CD_ORIGEM_DESPESA                                                   20140000   
DS_ORIGEM_DESPESA                        Publicidade por materiais impressos   
SQ_DESPESA                                                          224

Prosseguindo a análise é de se estranhar o gasto da candidata JULIARINA APARECIDA DA CRUZ PEREIRA, houve muito gastos com publicidade e vários cabos eleitorais, para meros 478 votos, e um gasto total de R$270.604,20. Também é de se estranhar como a candidata deixou para gastar o dinheiro bem próximo às eleições, visto que a campanha eleitoral começa bem antes do que seu primeiro registro de gasto.

In [14]:
df1[df1["SQ_CANDIDATO"]==130000612288].sort_values(by=["VR_DESPESA_CONTRATADA"],ascending=0).T

189051  \
DT_GERACAO                                               2019-09-03 00:00:00   
HH_GERACAO                                                          00:55:37   
ANO_ELEICAO                                                             2018   
CD_TIPO_ELEICAO                                                            2   
NM_TIPO_ELEICAO                                                    Ordinária   
CD_ELEICAO                                                               297   
DS_ELEICAO                                        Eleição Geral Federal 2018   
DT_ELEICAO                                                        07/10/2018   
ST_TURNO                                                                   1   
TP_PRESTACAO_CONTAS                                                    Final   
DT_PRESTACAO_CONTAS                                               06/11/2018   
SQ_PRESTADOR_CONTAS                                                422496932   
SG_UF                                                                     MG   
SG_UE                                                                     MG   
NM_UE                                                           MINAS GERAIS   
NR_CNPJ_PRESTADOR_CONTA                                       31209692000106   
CD_CARGO                                                                   7   
DS_CARGO                                                   Deputado Estadual   
SQ_CANDIDATO                                                    130000612288   
NR_CANDIDATO                                                           20610   
NM_CANDIDATO                          MOAMED RACHID GARIFF NOLETO DE QUEIROZ   
NR_CPF_CANDIDATO                                                 53918231615   
NR_CPF_VICE_CANDIDATO                                                     -1   
NR_PARTIDO                                                                20   
SG_PARTIDO                                                               PSC   
NM_PARTIDO                                            Partido Social Cristão   
CD_TIPO_FORNECEDOR                                                         1   
DS_TIPO_FORNECEDOR                                           Pessoa Jurídica   
CD_CNAE_FORNECEDOR                                                     58191   
DS_CNAE_FORNECEDOR         Edição de cadastros, listas e outros produtos ...   
NR_CPF_CNPJ_FORNECEDOR                                        22220898000146   
NM_FORNECEDOR                                   ML SERVIÇOS GRAFICOS LTDA ME   
NM_FORNECEDOR_RFB                                  ML SERVICOS GRAFICOS LTDA   
CD_ESFERA_PART_FORNECEDOR                                                 -1   
DS_ESFERA_PART_FORNECEDOR                                             #NULO#   
SG_UF_FORNECEDOR                                                      #NULO#   
CD_MUNICIPIO_FORNECEDOR                                                   -1   
NM_MUNICIPIO_FORNECEDOR                                               #NULO#   
SQ_CANDIDATO_FORNECEDOR                                                   -1   
NR_CANDIDATO_FORNECEDOR                                                   -1   
CD_CARGO_FORNECEDOR                                                       -1   
DS_CARGO_FORNECEDOR                                                   #NULO#   
NR_PARTIDO_FORNECEDOR                                                     -1   
SG_PARTIDO_FORNECEDOR                                                 #NULO#   
NM_PARTIDO_FORNECEDOR                                                 #NULO#   
DS_TIPO_DOCUMENTO                                                Nota Fiscal   
NR_DOCUMENTO                                                         185 - A   
CD_ORIGEM_DESPESA                                                   20140000   
DS_ORIGEM_DESPESA                        Publicidade por materiais impressos   
SQ_DESPESA                                                          22

Já o candidato MOAMED RACHID GARIFF NOLETO DE QUEIROZ teve a maioria dos seus gastos concentrados nos dias 27/08/2018 e 28/08/2018, o que é de se chamar certa atenção pois não é um padrão de gasto comum, talvez este candidato tenha tido muito planejamento a priori, porém obteve 707 votos gastando R$253.626,40, e um custo por voto de aproximadamente 360 reais

In [24]:
df1[df1["SQ_CANDIDATO"]==130000610154].sort_values(by=["VR_DESPESA_CONTRATADA"],ascending=0).T

171011  \
DT_GERACAO                                        2019-09-03 00:00:00   
HH_GERACAO                                                   00:55:37   
ANO_ELEICAO                                                      2018   
CD_TIPO_ELEICAO                                                     2   
NM_TIPO_ELEICAO                                             Ordinária   
CD_ELEICAO                                                        297   
DS_ELEICAO                                 Eleição Geral Federal 2018   
DT_ELEICAO                                                 07/10/2018   
ST_TURNO                                                            1   
TP_PRESTACAO_CONTAS                                             Final   
DT_PRESTACAO_CONTAS                                        13/11/2018   
SQ_PRESTADOR_CONTAS                                         422497104   
SG_UF                                                              MG   
SG_UE                                                              MG   
NM_UE                                                    MINAS GERAIS   
NR_CNPJ_PRESTADOR_CONTA                                31208237000196   
CD_CARGO                                                            7   
DS_CARGO                                            Deputado Estadual   
SQ_CANDIDATO                                             130000610154   
NR_CANDIDATO                                                    51003   
NM_CANDIDATO                                     ELANE CRISTINA ALVES   
NR_CPF_CANDIDATO                                           4470911666   
NR_CPF_VICE_CANDIDATO                                              -1   
NR_PARTIDO                                                         51   
SG_PARTIDO                                                      PATRI   
NM_PARTIDO                                                   Patriota   
CD_TIPO_FORNECEDOR                                                  1   
DS_TIPO_FORNECEDOR                                    Pessoa Jurídica   
CD_CNAE_FORNECEDOR                                              49248   
DS_CNAE_FORNECEDOR                                 Transporte escolar   
NR_CPF_CNPJ_FORNECEDOR                                 16694831000140   
NM_FORNECEDOR                     KEYNES AUGUSTO DE SOUZA 03273783605   
NM_FORNECEDOR_RFB                 KEYNES AUGUSTO DE SOUZA 03273783605   
CD_ESFERA_PART_FORNECEDOR                                          -1   
DS_ESFERA_PART_FORNECEDOR                                      #NULO#   
SG_UF_FORNECEDOR                                               #NULO#   
CD_MUNICIPIO_FORNECEDOR                                            -1   
NM_MUNICIPIO_FORNECEDOR                                        #NULO#   
SQ_CANDIDATO_FORNECEDOR                                            -1   
NR_CANDIDATO_FORNECEDOR                                            -1   
CD_CARGO_FORNECEDOR                                                -1   
DS_CARGO_FORNECEDOR                                            #NULO#   
NR_PARTIDO_FORNECEDOR                                              -1   
SG_PARTIDO_FORNECEDOR                                          #NULO#   
NM_PARTIDO_FORNECEDOR                                          #NULO#   
DS_TIPO_DOCUMENTO                                         Nota Fiscal   
NR_DOCUMENTO                                               000038 - A   
CD_ORIGEM_DESPESA                                            20600000   
DS_ORIGEM_DESPESA                       Cessão ou locação de veículos   
SQ_DESPESA                                                   23721672   
DT_DESPESA                                                 24/09/2018   
DS_DESPESA                 LOCAÇÃO DE VANS DE 24/09/2018 A 06/10/2018   
VR_DESPESA_CONTRATADA                                            9100   

                                                                  170906  \
DT_GERACAO                                  

In [33]:
df1[df1["SQ_CANDIDATO"]==130000610154]["DS_DESPESA"].value_counts()

SERVIÇOS DE PANFLETAGEM E DIVULGAÇÃO DE CAMPANHA    258
TAR FL CHQ                                            9
GASOLINA COMUM                                        3
MOTORISTA                                             2
COORDENADOR DE CAMPANHA                               2
DIESEL S10                                            2
ETANOL COMUM                                          2
GASOLINA V POWER                                      2
IMPRESSOS SANTINHOS 7 X 10                            2
LOCAÇÃO DE VANS DE 24/09/2018 A 06/10/2018            1
SERVIÇOS DE MOTORISTA                                 1
COORDENADOR REGIONAL DE CAMPANHA                      1
COORDENADORA DE PANFLETAGEM                           1
IMPRESSOS PLACAS DE TRANSITO                          1
PÃO DE SAL E DOCE                                     1
IMPRESSOS PLACAS 50 X 70                              1
REFEIÇÃO                                              1
IMPRESSOS COLINHA                               

Já a candidata ELANE CRISTINA ALVES espalhou bem seus gastos, por exemplo, contratou 258 distribuidores de panfletos, para obter 336 votos e um gasto total de R$228.111,00.

Concluindo: <br />

Nesta análise pegamos os 3 candidatos da lista possíveis laranjas com maiores gastos para ver seus gastos mais de perto<br />

A candidata ELANE CRISTINA ALVES obteve o custo por voto mais impressionante, cerca de 679 reais por voto, o que equivale a praticamente 20 vezes mais que o maior custo por voto de um candidato eleito, de 34 reais.<br />
Esta candidata também chama atenção por ter ensino fundamental incompleto num partido onde isso é incomum e por ser mulher, uma característica assustadoramente entra candidatos laranjas<br />

Já o candidato MOAMED RACHID GARIFF NOLETO DE QUEIROZ, obteve um custo por voto de aproximademente 359 reais. <br />

E a candidata JULIARINA APARECIDA DA CRUZ PEREIRA 566 reais aproximademente. <br />

Infelizmente é inconclusivo se estes candidatos realmente são laranjas, talvez se fosse pego as notas fiscais e feito as devidas diligências. Encerro aqui a análise. <br />